lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -1,11 +1,13 @@
 <%page expression_filter="h"/>
 <%inherit file="/main.html" />
 <%namespace name='static' file='/static_content.html'/>
-<%def name="online_help_token()"><% return "instructor" %></%def>
 <%!
 from django.utils.translation import ugettext as _
 from django.core.urlresolvers import reverse
+from django.conf import settings
 %>
+<%def name="online_help_token()"><% return "instructor" %></%def>
+<%def name="doc_base_url()"><% return settings.INSTRUCTOR_DOC_LINK_BASE_URL %></%def>
```

Comment:
Oh, would it work this way: I add other online help tokens to the other template files in the instructor_dashboard folder, also, adding this bit ```<%def name="doc_base_url()"><% return settings.INSTRUCTOR_DOC_LINK_BASE_URL %></%def>```. I then have to add any other online help token definitions to cms_config.ini?